medicines/api/src/substance.rs

```diff
@@ -22,6 +27,7 @@ pub async fn get_substances(first: i32) -> Substances {
         .take(first as usize)
```

Comment:
Is this stubbed out 1000 ibuprofens something that needs replacing before this goes to prod?

Reply:
as part of #403